components/scg/scg.html

```diff
@@ -102,13 +102,13 @@
         <script type="text/javascript" src="src/scg-component.js"></script>
         <script type="text/javascript" src="src/listener/scg-mode-bus.js"></script>
         <script type="text/javascript" src="src/listener/scg-mode-contour.js"></script>
-        <script type="text/javascript" src="src/listener/scg-mode-edge.js"></script>
+        <script type="text/javascript" src="src/listener/scg-mode-connector.js"></script>
         <script type="text/javascript" src="src/listener/scg-mode-link.js"></script>
 		<script type="text/javascript" src="src/listener/scg-mode-select.js"></script>
 		<script type="text/javascript" src="src/command/append-object.js"></script>
         <script type="text/javascript" src="src/command/command-manager.js"></script>
         <script type="text/javascript" src="src/command/create-node.js"></script>
-        <script type="text/javascript" src="src/command/create-edge.js"></script>
+        <script type="text/javascript" src="src/command/create-connector.js"></script>
         <script type="text/javascript" src="src/command/create-bus.js"></script>
         <script type="text/javascript" src="src/command/create-contour.js"></script>
         <script type="text/javascript" src="src/command/create-link.js"></script>
```

Comment:
commands use create and delete

Reply:
All sc.g-commands don't generate anything in the sc-memory, they create sc.g-elements on the scene